.changeset/six-mangos-hide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystatic/core': patch
+'@keystar/ui': patch
+---
+
+Support `ui.navigation` config. Fix `StatusLight` passing through "className" to DOM element.
```

Comment:
Doesn't matter much but just fyi, you can create multiple changesets for a single PR so e.g. here you could have one changeset for `ui.navigation` and another for the `StatusLight` fixes